parser/parser_test.go

```diff
@@ -82,21 +82,6 @@ func TestFromFile(t *testing.T) {
 				assert.NotNil(t, w.States[0].(*model.OperationState).Actions[0].FunctionRef)
 				assert.Equal(t, "greetingFunction", w.States[0].(*model.OperationState).Actions[0].FunctionRef.RefName)
 			},
-		}, {
-			"./testdata/workflows/greetings-v08-spec.sw.yaml",
-			func(t *testing.T, w *model.Workflow) {
-				assert.Empty(t, w.Name)
-				assert.Empty(t, w.Start)
-				assert.IsType(t, &model.OperationState{}, w.States[0])
-				assert.Equal(t, "custom.greeting", w.ID)
-				assert.NotEmpty(t, w.States[0].(*model.OperationState).Actions)
-				assert.NotEmpty(t, w.Functions[0])
-				assert.Equal(t, "greetingCustomFunction", w.Functions[0].Name)
-				assert.Equal(t, model.FunctionTypeCustom, w.Functions[0].Type)
-				assert.NotNil(t, w.States[0].(*model.OperationState).Actions[0].FunctionRef)
-				assert.Equal(t, "greetingCustomFunction", w.States[0].(*model.OperationState).Actions[0].FunctionRef.RefName)
-				assert.Equal(t, "greetingCustomFunction", w.States[0].(*model.OperationState).Actions[0].Name)
-			},
 		}, {
```

Comment:
This test is not used anymore?

Reply:
I moved it to the last item in this array and added more checks there.